### PR TITLE
Add Rari claim fees

### PR DESCRIPTION
**Pull request by graszka22**

### CODE REVIEW

File: src/actions/transactionEstimateActions.js [hunk 1]
```diff
@@ -64,6 +64,12 @@ export const setEstimatingTransactionAction = (isEstimating: boolean) => ({
   payload: isEstimating,
 });
 
+export const setEstimatingErrorAction = (errorMessage: string) => {
+  return (dispatch: Dispatch) => {
+    dispatch({ type: SET_TRANSACTION_ESTIMATE_ERROR, payload: errorMessage });
+  };
```
Comment (IAmKio) [inline src/actions/transactionEstimateActions.js:70]: Could be:

```suggestion
  return (dispatch: Dispatch) => dispatch({ type: SET_TRANSACTION_ESTIMATE_ERROR, payload: errorMessage });
```

... maybe?

File: src/screens/Rari/RariClaimRgtReview.js [hunk 2]
```diff
@@ -65,10 +66,28 @@ const RariClaimRgtReview = ({
...
+        Toast.show({
+          message: t('toast.rariFetchDataFailed'),
+          emoji: 'hushed',
+          supportLink: true,
+          autoClose: false,
+        });
+      }
+      setClaimFee(fee);
+      setIsFetchingFee(false);
+    }).catch(() => null);
```
Comment (IAmKio) [inline src/screens/Rari/RariClaimRgtReview.js:85]: Just wondering what happens here from a UX perspective? I think the returning of `null` is what is throwing me off...

File: src/utils/rari.js [hunk 3]
```diff
@@ -837,12 +837,35 @@ export const getWithdrawalFeeRate = (rariPool: RariPool) => {
     });
 };
 
-export const getRariClaimRgtTransaction = (senderAddress: string, amount: number, txFeeInWei?: BigNumber) => {
+export const getClaimRtgFee = async () => {
+  const distributionStartBlock = 11094200;
+  const distributionPeriod = 390000;
+  const distributionEndBlock = distributionStartBlock + distributionPeriod;
+  const initialClaimFee = 0.33;
```
Comment (IAmKio) [inline src/utils/rari.js:844]: Just wondering, do any of these need to go into our envConfig / remote config?

```
  const distributionStartBlock = 11094200;
  const distributionPeriod = 390000;;
  const initialClaimFee = 0.33;
```

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few questions but great progress all in all! 👍

File: src/utils/rari.js (hunk 3 above)
Comment (graszka22) [reply src/utils/rari.js:844]: No, because they're not going to change. Unless there is another liquidity mining program :D

Comment (IAmKio) [reply src/utils/rari.js:844]: Haha, fair! Thanks for clearing that up.

**Review by IAmKio — APPROVED** [review]

**Pull request merged**